docs/proposals/apiman-integration.md

```diff
@@ -0,0 +1,123 @@
+# APIMan Integration
+
+## Abstract
+This proposes a design for integrating [APIMan](http://www.apiman.io/) with OpenShift in order to provide  micro-services governance for API service providers.
+
+## Motivation
+Providers of services may have the need to control who and how their services are consumed.  This may be realized as policies defined by providers to control a service
+in any number of ways (e.g security, throttling/quota, billing and metrics)<sup>[1](#r1)</sup>. 
+
+## Constraints and Assumptions
+### Authentication
+The OAuth token identifying a user in OpenShift will be utilized by APIMan to associate service policy to their identity.  This will ensure a user's identity is 
+ubiquitous across the cluster when interacting with OpenShift and the APIMan management console.
+
+### Deployment
+#### Communication
+APIMan Components (e.g. gateway, management interface) integrated with the cluster will utilize mutual TLS for communication.
+
+#### On-Cluster Policy Storage
+[Origin Aggregated logging](https://github.com/openshift/origin-aggregated-logging)<sup>[4](#r4)</sup> will be deployed before APIMan in order to utilize the existing ElasticSearch cluster for policy storage.  
```

Comment:
As mentioned yesterday this is a dubious approach for a variety of reasons. APIMan can support other backends. For configuration it needs a clustered DB for HA, not necessarily ES - could be postgres/mysql if we have an easy way to cluster those, or maybe mongodb (not sure if Hibernate supports that). For metrics reporting we should explore sending them to Hawkular instead, which is an existing RFE.